custom/enikshay/duplicate_ids.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from collections import Counter
+import six
+
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from dimagi.utils.couch.database import iter_docs
+
+from .const import VOUCHER_ID, ENIKSHAY_ID
+
+
+def get_cases_with_duplicate_ids(domain, case_type, all_case_ids):
```

Comment:
Doc comments on these functions will be useful for others.

Reply:
I just added a file-level explanation of the issue that these functions address.  With that context, my hope is that the function names can be descriptive enough.